client/src/menu/game/gameScreenContent/WillMenu.tsx

```diff
@@ -24,6 +26,7 @@ export default class WillMenu extends React.Component<{}, WillMenuState> {
                 will: GAME_MANAGER.state.will,
                 notes: GAME_MANAGER.state.notes,
                 deathNote: GAME_MANAGER.state.deathNote,
+                cantPost: GAME_MANAGER.state.sendChatGroups.length !== 0,
```

Comment:
This is inverted

Reply:
You can post your alibi as long as you have at least one sendChatGroup

Reply:
i changed this to ===